src/main/java/org/gooru/insights/services/BaseConnectionServiceImpl.java

```diff
@@ -309,6 +314,21 @@ private void fieldArrayHandler(){
 		}
 	}
 	
+	private void exportConfig() {
+		
+		exportFieldCache = new HashMap<String, String>();
+		exportReportCache = new HashMap<String, String>();
+		
+		ColumnList<String> columnList = baseCassandraService.read(CassandraConstants.Keyspaces.INSIGHTS.keyspace(), CassandraConstants.ColumnFamilies.CONFIG_SETTINGS.columnFamily(), CassandraConstants.CassandraRowKeys.EXPORT_FIELDS.CassandraRowKey());
```

Comment:
looks like both the iteration logic doing the same stuff, Can we merge it together ?